[INFO   ] Logger: Record log in /storage/emulated/0/Download/Python/.kivy/logs/kivy_24-04-13_77.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 190 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Download/Python/main.py", line 4, in <module>
[WARNING] stderr:     from game import PongGame
[WARNING] stderr:   File "/storage/emulated/0/Download/Python/game.py", line 74
[WARNING] stderr:     else if(self.player2.score == maxValue):
[WARNING] stderr:          ^^
[WARNING] stderr: SyntaxError: expected ':'
